datalad/tests/utils.py

```diff
@@ -231,6 +235,14 @@ def newfunc(*arg, **kw):
                 # well -- if something wrong just proceed with defaults
                 pass
 
+        # if DATALAD_TESTS_TEMPDIR is set, use that as directory,
+        # let mktemp handle it otherwise. However, an explicitly provided
+        # dir=... will override this.
+        directory = os.environ.get('DATALAD_TESTS_TEMPDIR')
```

Comment:
that omission was "non intensional" ;)  indeed -- please add it, along with nose .  For now imho the more the merrier since that requirements is primarily used for setting up testing environments